src/test/java/cloud/fogbow/ras/core/OrderControllerTest.java

```diff
@@ -420,75 +597,85 @@ public void testDeleteNullOrder()
         this.ordersController.deleteOrder(null);
     }
 
-    // test case: Getting an order with an unexisting id must throw an InstanceNotFoundException
+    // test case: Getting an order with a nonexistent id must throw an InstanceNotFoundException
     @Test(expected = InstanceNotFoundException.class)
     public void testGetOrderWithInvalidId() throws InstanceNotFoundException {
-        this.ordersController.getOrder("invalid-order-id");
+        this.ordersController.getOrder(INVALID_ORDER_ID);
     }
-
-    private String getComputeOrderCreationId(OrderState orderState) throws InvalidParameterException, UnexpectedException {
-        SystemUser systemUser = new SystemUser("fake-id", "fake-user", "fake-token-provider");
-
-        ComputeOrder computeOrder = Mockito.spy(new ComputeOrder());
+    
+    private NetworkOrder createFulfilledNetworkOrder(SystemUser systemUser) {
+        NetworkOrder networkOrder = new NetworkOrder();
+        networkOrder.setSystemUser(systemUser);
+        networkOrder.setRequester(BaseUnitTests.LOCAL_MEMBER_ID);
+        networkOrder.setProvider(BaseUnitTests.LOCAL_MEMBER_ID);
+        networkOrder.setOrderStateInTestMode(OrderState.FULFILLED);
+        return networkOrder;
+    }
+    
+    private ComputeOrder createFulfilledComputeOrder(SystemUser systemUser) {
+        ComputeOrder computeOrder = new ComputeOrder();
         computeOrder.setSystemUser(systemUser);
-        computeOrder.setRequester(this.localMember);
-        computeOrder.setProvider(this.localMember);
+        computeOrder.setRequester(BaseUnitTests.LOCAL_MEMBER_ID);
+        computeOrder.setProvider(BaseUnitTests.LOCAL_MEMBER_ID);
+        computeOrder.setOrderStateInTestMode(OrderState.FULFILLED);
+        return computeOrder;
+    }
+    
+    private InstanceStatus createInstanceStatus(ComputeOrder computeOrder) throws InstanceNotFoundException {
+        return new InstanceStatus(computeOrder.getId(),
+                computeOrder.getProvider(), computeOrder.getCloudName(),
+                InstanceStatus.mapInstanceStateFromOrderState(computeOrder.getOrderState()));
+    }
+
+    private String getComputeOrderCreationId(OrderState orderState)
```

Comment:
I think this name could be improved. This method does more than just get the order id, right? Besides, creationId? Wouldn't it be just id?

Reply:
I'll fix this.